rfcs/0119-testing-conventions.md

```diff
@@ -0,0 +1,114 @@
+---
+feature: Defined conventions around testing of official Nixpkgs packages.
+start-date: 2021-09-03
```

Comment:
oops, been dragging my feet on this for a while...
```suggestion
start-date: 2021-29-12
```

Reply:
This should be YYYY-MM-DD instead

Reply:
Yep, sorry. American brain.